src/gwt/src/org/rstudio/studio/client/workbench/ui/PaneManager.java

```diff
@@ -579,6 +579,14 @@ public void onFocusCenterSeparator()
       panel_.focusSplitter();
    }
 
+   @Handler
```

Comment:
Should we show an alert if command is invoked and there are already MAX_COLUMN_COUNT columns?

Or, could instead disable the command when there are that many columns, and it will thus be disabled in the menu and the command-palette. Might be simpler just to keep enabled and show an alert.

Reply:
Ok, I updated to show an alert when MAX_COLUMN_COUNTS is reached and when the `Allow Source Columns` pref is set to false.